assembly/src/docs/20_INSTALL_WINDOWS_WITH_GRAALVM.md

```diff
@@ -0,0 +1,133 @@
+# Build TornadoVM with Graal 20.2.0 JDK 8 / 11 on Windows x64
+## Install tools
+### Maven
+Download Apache Maven from the [official site](https://maven.apache.org/download.cgi) and extract it to the any location. Below it's assumed that Maven's home is `C:/Maven`.
+### MSys2
+1. Download [MSys2](https://www.msys2.org/) installer from the official site and run it. You may choose any installation directory, below it will be referred as `<MSYS2>`. IMPORTANT: the only executable you should use as a terminal is `<MSYS2>/mingw64.exe`, don't use others (msys2.exe, mingw32.exe) as this would lead to unpredictable results.
```

Comment:
`Download the [MSys2](https://www.msys2.org/) installer  .. `

Remove this part:
`don't use others (msys2.exe, mingw32.exe) as this would lead to unpredictable results.`.

Reply:
```suggestion
1. Download [MSys2](https://www.msys2.org/) installer from the official site and run it. You may choose any installation directory, below it will be referred as `<MSYS2>`. IMPORTANT: the only executable you should use as a terminal is `<MSYS2>/mingw64.exe`.
```